src/CorsListener.php

```diff
@@ -47,14 +47,15 @@ public function onKernelResponse(ResponseEvent $event): void
             https://github.com/symfony/http-foundation/blob/5.1/Response.php#L290
             to make the response deterministic, set the content type explicitly */
             $response->headers->set('Content-Type', 'text/html; charset=UTF-8');
-            /* no content -> no cache control needed, but it's set automatically by symfony in
-            https://github.com/symfony/http-foundation/blob/5.1/ResponseHeaderBag.php#L133) */
-            $response->headers->remove('Cache-Control');
```

Comment:
To, ze `OPTIONS` nema body, neznamena, ze se neda cachovat - cachuji se i headers. Byl s tim defaultem ze Symfony nejaky problem?

Jinak to, ze `Cache-Control` neni nastaveny neznamen "necachovat", ale "kliente, udelej si co chces".
> If you don't add a Cache-Control header because the response is not intended to be cached, that could cause an unexpected result. Cache storage is allowed to cache it heuristically — so if you have any requirements on caching, you should always indicate them explicitly, in the Cache-Control header.

(https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#up-to-date_contents_always)

Reply:
> Byl s tim defaultem ze Symfony nejaky problem?

Na to už ti asi nedokážu odpovědět (jako je tam nalinkovaný symfony 5.1 - to je 4 roky :) ). Jediný, co mě napadá, je že by to pak mohlo chtít cachovat i ten následující request, ale fakt nevím. Maximálně se zeptat na UI jestli je nenapadne, čemu by to mohlo vadit